### PR TITLE
RSS-ECOMM-4_12: Implement "Remove from Cart" Button

### PR DESCRIPTION
**Pull request by SvetaIlina**
## Overview
implement "Remove from Cart" button

**What has been done:**
- add a "Remove from Cart" button  for each item in the shopping cart.
- integrate button with commercetools API for updating cart

**Why this has been done:**
- To improve user experience in managing their shopping cart 

**Issue(s) closed:**
#64 

## Type of Change
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Screenshots
image

## Checklist:
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: kickFlip-project/src/utils/kickflip-api.ts [hunk 1]
```diff
@@ -626,7 +626,7 @@ export const getProductImg = async (id: string, color: string) => {
     }
 };
 
-export const updateCartQuantitty = async (cartId: string, updateLineItemQuantity: ChangeLineItemQuantity) => {
+export const updateCart = async (cartId: string, updateLineItemQuantity: ChangeLineItem) => {
     const response = await fetch(`${URL}/${projectKey}/carts/${cartId}`, {
```
Comment (makarovaiuliia) [inline kickFlip-project/src/utils/kickflip-api.ts]: Лучше делать все запросы через me эндпоинт

**Review by makarovaiuliia — CHANGES_REQUESTED** [review]
Все работает! Надо чуть поправить запрос, чтоб все было по одному эндпоинту

File: kickFlip-project/src/utils/kickflip-api.ts (hunk 1 above)
Comment (SvetaIlina) [reply kickFlip-project/src/utils/kickflip-api.ts]: готово

**Review by makarovaiuliia — APPROVED** [review]
супер!

**Pull request merged**